src/AutoLoginMiddleware.php

```diff
@@ -20,17 +20,20 @@ final class AutoLoginMiddleware implements MiddlewareInterface
     private IdentityRepositoryInterface $identityRepository;
     private LoggerInterface $logger;
     private AutoLogin $autoLogin;
+    private bool $addCookie;
 
     public function __construct(
         User $user,
         IdentityRepositoryInterface $identityRepository,
         LoggerInterface $logger,
-        AutoLogin $autoLogin
+        AutoLogin $autoLogin,
+        bool $addCookie = true
```

Comment:
Looks like we should not introduce the option and should not set cookie in the middleware.

Docs should be added to guide user about adding the cookie.

Reply:
Save `$addCookie` for cases when always need set cookie, but default value is `false`.
Guide to readme added.
Also adopt to psalm.